src/modules/user/user.module.ts

```diff
@@ -1,10 +1,16 @@
 import { Module } from "@nestjs/common";
 import { UserQueriesController } from "src/modules/user/api/user.queries.controller";
 import { UserMutationsController } from "src/modules/user/api/user.mutations.controller";
+import { UserService } from "src/modules/user/domain/user.service";
+import { UserServiceName } from "src/modules/user/domain/user.service.types";
+import { UserRepository } from "src/modules/user/repository/user.repository";
+import { TypeOrmModule } from "@nestjs/typeorm";
+import { UserRecord } from "src/core/infrastructure/entities/User";
 
 @Module({
-	imports: [],
+	imports: [TypeOrmModule.forFeature([UserRecord])],
```

Comment:
This is where we define the Repository tables the module has access to. So the users module only has access to the User Record right now.